src/open_producten/producten/models/eigenaar.py

```diff
@@ -0,0 +1,97 @@
+from django.core.exceptions import ValidationError
+from django.core.validators import MinLengthValidator, RegexValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from open_producten.producten.models.validators import validate_bsn
+from open_producten.utils.models import BaseModel
+
+from .product import Product
+
+
+class Eigenaar(BaseModel):
+
+    product = models.ForeignKey(
+        Product,
+        verbose_name=_("product"),
+        related_name="eigenaren",
+        on_delete=models.CASCADE,
+        help_text=_("De organisatie van het contact"),
+    )
+
+    bsn_nummer = models.CharField(
+        _("BSN nummer"),
```

Comment:
Is there a specific reason this was renamed to `bsn_nummer`?
```suggestion
    bsn = models.CharField(
        _("Burgerservicenummer"),
```